car.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'transport'
+
+class Car < Transport
+  attr_accessor :number
+
+  def initialize(weight = 100, speed = 50, available = true, distance = 400)
```

Comment:
Here and in the `Bike` class you should use constants as a replacement of "magic numbers" which are appear to be your speed, weight and distance definitions in this class.

You can require constants module in this file and do something like this: 
```
@weight = Constants::BIKE_MAX_WEIGHT
```

Reply:
Done. I use constants as a definition parameters, not int he definition body. Otherwise I need to input parameters while running. I made like this to simplify testing the code

Reply:
`  def initialize(
    weight = Constants::CAR_MAX_WEIGHT,
    speed = Constants::CAR_MAX_SPEED,
    available = true,
    distance = Constants::CAR_DEFAULT_DISTANCE
  )
    @number = "#{rand(9999)}AR"
    super
  end`

Is it considered to be good style?

Reply:
Done. Thanks